src/systemctl/sysv-compat.h

```diff
@@ -3,7 +3,10 @@
 
 #include "time-util.h"
 
+#if HAVE_SYSV_COMPAT
 int talk_initctl(char runlevel);
+#endif
```

Comment:
i think it's generally a good idea to hide ifdeffery in .c files and keep it out of .h files, hence I'd probably not bother with this. Yes, this would mean we define an empty stub process if sysv compat is off which we don't use, but the linker should be smart enough to strip that really...

(but doesn't matter really)

Reply:
I agree in general, but in this case the function has just one user, and the argument is calculated by another function guarded by ifdef. So if call site has to be under `#ifdef`, which means that the stub function wouldn't be used anywhere. So in the end, I think it's better to it this way.